.github/workflows/build-kata-static-tarball-amd64.yaml

```diff
@@ -27,6 +27,7 @@ jobs:
           - nydus
           - qemu
           - qemu-tdx-experimental
+          - qemu-snp-experimental
```

Comment:
@fitzthum, we're trying to keep those alphabetically sorted. Mind to move this entry before the qemu-tdx-experimental?